tenseal/binding.cpp

```diff
@@ -131,6 +131,8 @@ PYBIND11_MODULE(_tenseal_cpp, m) {
              })
         .def("mul", &BFVVector::mul)
         .def("mul_", &BFVVector::mul_inplace)
+        .def("mul_plain", py::overload_cast<const int64_t &>(
+                              &BFVVector::add_plain, py::const_))
```

Comment:
This looks like a typo.
Please add some Python tests for the explicit mul_plain, to prevent this kind of bug.